ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -135,12 +152,12 @@ public LogicalSchema withPseudoAndKeyColsInValue(final boolean windowed) {
    * @return the new schema with the columns removed.
    */
   public LogicalSchema withoutPseudoAndKeyColsInValue() {
-    return rebuild(false, false);
+    return rebuild(false, false, -1);
```

Comment:
What does -1 mean? Should this be the current pseudocolumn version instead?

Reply:
The passed `pseudoColumnVersion` doesn't actually see use if the first boolean is false, as if we are building without pseudocolumns, the version number is irrelevant. It seems overloading the method would be better here? Though there may be a fair bit of repeat code with the overload.